pods/Numberick-CoreKit.podspec

```diff
@@ -0,0 +1,42 @@
+Pod::Spec.new do |s|
+    s.name             = 'Numberick-CoreKit'
+    s.version          = '999.99.9'
+    s.summary          = "A new protocol hierarchy that refines Swift's standard integer library."
+  
+    s.description      = <<-DESC
+    A new protocol hierarchy that refines Swift's standard library. Core part of Numberick.
+                         DESC
+  
+    s.homepage         = 'https://github.com/oscbyspro/Numberick'
+  
+    s.license          = { :type => 'Apache-2.0', :file => 'LICENSE' }
+    s.author           = { 'Oscar Byström Ericsson' => 'oscbyspro@protonmail.com' }
+    s.source           = { :git => 'https://github.com/oscbyspro/Numberick.git', :tag => "v#{s.version}" }
+  
+    base_platforms     = { :ios => '14.0', :osx => '11.0', :tvos => '14.0' }
+
+    s.platforms        = base_platforms.merge({ :watchos => '7.0' })
+
+    s.module_name      = 'NBKCoreKit'
+    
+    s.swift_version    = '5.7'
+    
+    s.source_files     = 'Sources/NBKCoreKit/**/*.swift'
+
+    swift_custom_flags = ''
+    
+    if ENV['NBK_FEATURE_StaticBigInt'] == 'true'
+        base_platforms     = { :ios => '16.4', :osx => '13.3', :tvos => '16.4' }
+        s.platforms        = base_platforms.merge({ :watchos => '9.4' })
+        s.swift_version    = '5.8'
+        swift_custom_flags += 'SBI '
+    end
+
+    s.pod_target_xcconfig = { 'SWIFT_ACTIVE_COMPILATION_CONDITIONS' => swift_custom_flags }
+    
+    s.test_spec 'Tests' do |ts|
+        ts.platforms = base_platforms
```

Comment:
Why is watchOS not added to test platforms?

Reply:
Because of the bug in the CocoaPods. They broke watchOS testing years ago and can't fix it till now. They fix it, they broke it, they fix it one more time. Seems like something with their integration with Xcode. I tried to enable it and it is broken now.

Reply:
https://github.com/CocoaPods/CocoaPods/issues/11558

Reply:
It's an old story. Apple implemented watchOS testing pretty late. CocoaPods were focused on testing on other platforms. When Apple released watchOS XCTest support CocoaPods quickly enabled it, but seems as if it should be handled differently. So it's buggy.